TmdbEasy/TmdbEasyClientv3.cs

```diff
@@ -19,28 +18,20 @@ public TmdbEasyClientv3(IJsonDeserializer jsonDeserializer, TmdbEasyOptions opti
 
             _options = options ?? throw new ArgumentNullException($"{nameof(TmdbEasyOptions)}");
 
-            _httpClient = new HttpClient()
-            {
-                BaseAddress = new Uri(_options.UseSsl ? UrlConstants.TmdbUrl3Ssl : UrlConstants.TmdbUrl3)
-            };
```

Comment:
Some reason setting the BaseAddress here doesn't take effect so i do it in Line 29 now.

Reply:
We need to add this back in. It works on master branch and should continue to do so here.

This should be the process:
1. Options are created and provided to the TmdbEasyClient.
2. The client is injectd into the requesthandler. 
3. Within the constructor of the requestHandler we should set the options (taken from client)
4. Each time a request is created, we should pass the options to the individual requests

Some way along the way something is being missed

Reply:
On master branch when the `Request` ctor was called it got the same value as the `_httpClient.BaseAddress` was obviously. Than when `await _httpClient.GetStringAsync(query)` was called the `query` parameter was the full URL so i think the `_httpClient.BaseAddress` was ignored. I will add back the BaseAddress initialization and [i leave this here](https://stackoverflow.com/questions/23438416/why-is-httpclient-baseaddress-not-working).

Reply:
I see what you mean. This can be simplified and was an oversight of mine. We are setting the base url on every request individually, so we dont need to set it on the http client at all.

Reply:
So this change should stay
``` diff
public Request(TmdbEasyOptions options)
{
-    _requestBuilder = new StringBuilder();
+    _requestBuilder = new StringBuilder(options.BaseUri);
     _options = options;
}
```
and the TmdbEasyClient should become

``` diff
public class TmdbEasyClientv3 : ITmdbEasyClient
    {
        private readonly IJsonDeserializer _jsonDeserializer;
        private readonly TmdbEasyOptions _options;
        private readonly HttpClient _httpClient;

        public TmdbEasyClientv3(IJsonDeserializer jsonDeserializer, TmdbEasyOptions options)
        {
            _jsonDeserializer = jsonDeserializer;

            _options = options ?? throw new ArgumentNullException($"{nameof(TmdbEasyOptions)}");

            _httpClient = new HttpClient();
        }

        public async Task<TmdbEasyModel> GetResponseAsync<TmdbEasyModel>(string query)
        {
            if (string.IsNullOrEmpty(query) || string.IsNullOrWhiteSpace(query))
                throw new ArgumentException($"{nameof(TmdbEasyClientv3)} query param null or empty");

-            query = _options.BaseUri + query;
            string jsonResult = await _httpClient.GetStringAsync(query);

            return _jsonDeserializer.DeserializeTo<TmdbEasyModel>(jsonResult);
        }

        public ApiVersion GetVersion() => ApiVersion.v3;
    }
```

You'll need to adjust the tests setup to reflect this as we have slightly changed the internals

Reply:
This is gonna be funny, because now that i rewriten most of the RequestTests cases i had to include the `https://api.themoviedb.org/3` as a variable but if You agree to use the HttpClient.BaseAddress solution intead i can remove that baked in variable.

Reply:
I guess it doesnt make a difference now, either way will work. Since you have done the work then we can stick with the HttpClient.BaseAddress solution. The `request.GetUri` method just wont return the absolute path, which I am ok with if you are.

Reply:
Thank You for the response i will add back the BaseAddress solution. One more question:
Can i move the `TmdbUrl3` and `TmdbUrl3Ssl` variables to the `TmdbEasyClientv3` as `private readonly static string`?

Reply:
Sure I can understand the reasoning 👍